src/golioth_rpc.c

```diff
@@ -17,7 +17,7 @@ LOG_TAG_DEFINE(golioth_rpc);
 // Request:
 //
 // {
-//      "id": "id_string,
+//      "id": "id_string",
```

Comment:
This is not related to "Add RPC unit test", so would preferably split it to separate commit.